src/CustomerBundle/Controller/DashboardController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\CustomerBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+
+final class DashboardController extends Controller
```

Comment:
shouldn't this be AbstractController?

Reply:
All contollers extends Contoller, it should be update in next major. See: [#610-comment](https://github.com/sonata-project/ecommerce/issues/610#issuecomment-562537867)